api/clients/config.go

```diff
@@ -0,0 +1,49 @@
+package clients
+
+import (
+	"fmt"
+	"time"
+)
+
+type EigenDAClientConfig struct {
+	// RPC is the HTTP provider URL for the Data Availability node.
+	RPC string
+
+	// The total amount of time that the client will spend waiting for EigenDA to confirm a blob
+	StatusQueryTimeout time.Duration
+
+	// The amount of time to wait between status queries of a newly dispersed blob
+	StatusQueryRetryInterval time.Duration
+
+	// The total amount of time that the client will waiting for a response from the EigenDA disperser
+	ResponseTimeout time.Duration
+
+	// The quorum IDs to write blobs to using this client. Should not include quorums 0 or 1.
+	CustomQuorumIDs []uint
+
+	// Signer private key in hex encoded format. This key should not be associated with an Ethereum address holding any funds.
+	SignerPrivateKeyHex string
+
+	// Whether to disable TLS for an insecure connection when connecting to a local EigenDA disperser instance.
+	DisableTLS bool
+
+	// The blob encoding version to use when writing blobs from the high level interface.
+	PutBlobEncodingVersion BlobEncodingVersion
+}
+
+var DefaultQuorums = map[uint]bool{0: true, 1: true}
```

Comment:
Not sure hardcoding this is a good idea. We may omit the custom quorum validation altogether since the same validation is already done by the disperser.

Reply:
The idea is to fail on startup vs when the first dispersal takes place. If this is invalid config then I think it's ok to encode into the client that it's invalid config.

Reply:
If we want to pull this off, I think it should read these from the source of truth (chain)

Reply:
Do you envision the default quorum IDs changing? What is the need for increased modularity here?